examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
This code doesn't really test anything (except that it run) which is why I wouldn't put under tests. To have proper tests for this is on the todo list. But still it is convenient to have the images available to check if everything is consistent or to get a clue what has to be changed if not. If we don't have a proper place for such code in the repo I'd simply keep it for me under an ignored directory.